code/__HELPERS/mobs.dm

```diff
@@ -176,19 +175,44 @@ Proc for attack log creation, because really why not
 
 	var/mob/living/living_target
 
-
 	if(target && isliving(target))
 		living_target = target
+	
+	var/hp =" "
+	if(living_target)
+		hp = "(NEWHP: [living_target.health])"
+
+	var/starget = "NON-EXISTENT SUBJECT"
+	if(target)
+		if(is_mob_target && target.ckey)
+			starget += "[target.ckey]"

```

Comment:
This will make starget `NON-EXISTENT SUBJECT ckey`. Plus, add some parenthesis to the ckey.